core/client/fs/src/main/java/alluxio/client/file/FileInStream.java

```diff
@@ -13,6 +13,7 @@
 
 import alluxio.Seekable;
 import alluxio.client.BoundedStream;
+import alluxio.client.CanUnbuffer;
```

Comment:
why is there the `alluxio.client.CanUnbuffer` instead of `org.apache.hadoop.fs.CanUnbuffer`

Reply:
There are no hadoop-related dependencies in `core-client-fs`. We need our own `CanUnbuffer` interface as if we are using `alluxio.Seekable` instead of `org.apache.hadoop.fs.Seekable`.

Reply:
okay, I got the point, that makes sense.